tests/layer_tests/tensorflow_tests/test_tf_Add.py

```diff
@@ -205,6 +312,20 @@ def test_add_placeholder_const_broadcast_3D(self, params, ie_device, precision,
                    ie_device, precision, ir_version=ir_version, temp_dir=temp_dir,
                    use_new_frontend=use_new_frontend, use_old_api=use_old_api)
 
+    # Extend test data by specific V2 cases
+    test_data_broadcast_3D_v2 = test_data_broadcast_3D.copy() + [
+        dict(x_shape=[1, 1, 1], y_shape=[1], y_value=np.inf),
+        dict(x_shape=[1, 1, 1], y_shape=[1], y_value=np.NINF)
+    ]
+    @pytest.mark.parametrize("params", test_data_broadcast_3D_v2)
+    @pytest.mark.nightly
+    def test_addv2_placeholder_const_broadcast_3D(self, params, ie_device, precision, ir_version, temp_dir,
+                                      use_new_frontend, use_old_api):
+        self._test(*self.create_add_placeholder_const_net(**params, ir_version=ir_version,
+                                                          use_new_frontend=use_new_frontend, use_addv2 = True),
+                   ie_device, precision, ir_version, temp_dir=temp_dir,
```

Comment:
please provide more interesting cases to test. Now we spend computing resources to test operation on tensors with one element.